### ISSUE TITLE
[Question]What features do you prefer to add to plugin in the near feature? 

### ISSUE THREAD

**Issue by Quorafind** [jmilldotdev/obsidian-idea-clock#1]
Would you mind feature requesting?

For example:

1. Showing content in Canvas?
2. Link to notes really?
3. Link to contents with simply drag and drop?

OR would you mind sharing roadmap?

**Comment by jmilldotdev** [comment]
![image](https://user-images.githubusercontent.com/33093632/140933990-1514ecf4-4a17-4edf-a51c-1ce204f5ad30.png)

here is a snapshot of what i plan to add in the near future. i am going to make linked notes a first class citizen, since that is what differentiates this from the already wonderful Excalidraw plugin.

the most important features are the four i have listed as non-core

**Comment by derekvan** [comment]
Would love also to have the ability to type on the canvas. That is, to create labels on the canvas (not new notes) that I could cluster notes around. 

At any rate, this plugin is very exciting. This kind of cork board is really a key feature I have been looking for in Obsidian. Thanks for sharing it!

**Comment by jmilldotdev** [comment]
Yeah, adding annotation cards is a planned feature, or cards with text on them that aren’t a link to another note.

thanks for the suggestion :)
